lib/PassPrinters/LLVMBuild.txt

```diff
@@ -0,0 +1,23 @@
+;===- ./lib/Passes/LLVMBuild.txt -------------------------------*- Conf -*--===;
+;
+;                     The LLVM Compiler Infrastructure
+;
+; This file is distributed under the University of Illinois Open Source
+; License. See LICENSE.TXT for details.
+;
+;===------------------------------------------------------------------------===;
+;
+; This is an LLVMBuild description file for the components in this subdirectory.
+;
+; For more information on the LLVMBuild system, please see:
+;
+;   http://llvm.org/docs/LLVMBuild.html
+;
+;===------------------------------------------------------------------------===;
+
+[component_0]
+type = Library
+name = PassPrinters
+parent = Libraries
+required_libraries = Passes Analysis Core IPA IPO InstCombine Scalar Support TransformUtils Vectorize
+
```

Comment:
For this file to have any effect, you need to add PassPrinters to the subdirectories list in the LLVMBuild.txt file in the lib/ directory above it.

The Linux linker failures were due to these dependencies being ignored on account of that.